src/main/resources/templates/index.html

```diff
@@ -17,11 +17,12 @@
             <h1 class="display-4">Welcome to BulkBuy</h1>
         </div>
     </div>
-</main>
 
-<div class="container">
+
+<div class="container outerContentWrapper">
     <!-- Stuff goes here eventually -->
 </div>
+</main>
```

Comment:
Moved the exit </main> tag to after the main content of the page. <main> represents the main content of the page.
https://www.w3schools.com/tags/tag_main.asp
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main

Reply:
Very good.